composer.json

```diff
@@ -1,24 +1,22 @@
 {
     "name": "jordanbrauer/unit-converter",
-    "type": "component",
     "description": "Convert standard units from one to another with this easy to use, lightweight package",
-    "version": "0.8.7",
     "license": "MIT",
+    "type": "library",
     "authors": [
         {
             "name": "jordanbrauer",
             "email": "18744334+jordanbrauer@users.noreply.github.com"
         }
     ],
     "require": {
-        "php": "^7.1 || ^8.0",
+        "php": "^7.3 || ~8.0.0 || ~8.1.0",
```

Comment:
`^8.0` is gone: sad, but necessary, given the massive amount of BC breaks introduced at each minor PHP version.

This means new release in november 2022, but I hope that's not too much pressure :D

Reply:
(at least the new release will have explicitly stated PHP 8.2 support in the CI pipeline, **before** a release)